app/controllers/api/v1/passwords_controller.rb

```diff
@@ -0,0 +1,38 @@
+module Api
+  module V1
+    class PasswordsController < DeviseTokenAuth::PasswordsController
+      def edit
+        # if a user is not found, return nil
```

Comment:
I think that all this actions are executed by devise token auth, i suggest not redefine this edit

Reply:
The idea in this "edit" is only return a Json, but the functionality is the same.

Reply:
Make senses